spec/models/planting_spec.rb

```diff
@@ -148,12 +151,6 @@ def one_hundred_day_old_planting
     planting.owner.should be_an_instance_of Member
   end
 
-  it "owner isn't necessarily the garden owner" do
```

Comment:
This test hints that plantings in someone else's garden were an allowed feature... not sure why.